lib/operations/connect.js

```diff
@@ -586,16 +590,16 @@ function transformUrlOptions(_object) {
     object.dbName = _object.defaultDatabase;
   }
 
-  if (object.maxpoolsize) {
-    object.poolSize = object.maxpoolsize;
+  if (object.maxPoolSize) {
```

Comment:
Why are these changing to camel case?

Reply:
At some point, we implementing the name normalizing logic in the new url parser that converts the values from lowercase back to camelCase. These need to be able to pull the camelCase values.